integration/mediation-tests/tests-common/integration-test-utils/src/main/java/org/wso2/esb/integration/common/utils/ESBIntegrationTest.java

```diff
@@ -292,6 +292,12 @@ protected void addInboundEndpoint(OMElement inboundEndpoint) throws Exception {
 		esbUtils.addInboundEndpoint(contextUrls.getBackEndUrl(), sessionCookie, inboundEndpoint);
 	}
 
+	protected boolean isInboundEndpointDeployed(OMElement inboundEndpoint) throws Exception {
+
```

Comment:
Shall we remove this extra line?

Reply:
I use wso2 style guide and it inserts a space after a method name.

Reply:
Maybe that is not intended. This especially makes single line methods fat. Shall we remove it?

Reply:
is that violates the wso2 coding standard? I was instructed follow wso2 coding guideline during my fast track training program.